.gitignore

```diff
@@ -16,3 +16,6 @@
 /tmp
 /public/system
 .env.local
+
+
+.env.local~
```

Comment:
This is good! I bet this will help other people not add this file to their git diffs accidentally.